### ISSUE TITLE
Use CmsTextFieldBundle for text field image views

### ISSUE THREAD

**Issue by UlrichKu** [chameleon-system/chameleon-system#21]
**Actual**
The core delivers pkgCmsTextFieldImage.html.twig (and ImageResponsive.html.twig) which must be present in the used theme then.

**Desired**
Now the bundle CmsTextFieldBundle can be registered and be listed as snippet path for all themes (after core).

**Comment by UlrichKu** [comment]
(Proposal was originally by @pixeljunkie)
Yes: Cleaner file structure was the idea I think.

**Comment by pixeljunkie** [comment]
Yes... the file is not core related, but part of the bundle. It should be part of the right bundle.

**Comment by pixeljunkie** [comment]
Is there a PR for this? I see no PR connected to the issue.

**Comment by UlrichKu** [comment]
> Is there a PR for this? I see no PR connected to the issue.

The feature is still being discussed here. (??)
